recoverytool/src/main.rs

```diff
@@ -174,6 +185,8 @@ async fn main() {
                 .flat_map(|res| {
                     let (_, SignedEpochOutcome { outcome, .. }) = res.expect("DB error");
 
+                    dbg!(outcome.epoch);
```

Comment:
I thought that git checks nag about it now.

Reply:
Right, need to squash the fixup and remove these at the same time

Reply:
Fixed